src/pytest/test.py

```diff
@@ -1890,6 +1890,14 @@ def testIssue484(env):
     expected = [6, ['value', 'white', 'value_count', '2'], ['value', 'cars', 'value_count', '2'], ['value', 'small cars', 'value_count', '1'], ['value', 'blue', 'value_count', '2'], ['value', 'Big cars', 'value_count', '2'], ['value', 'green', 'value_count', '1']]
     env.assertEqual(res, expected)
 
+def testIssue501(env):
+    env.cmd('FT.CREATE', 'incidents', 'SCHEMA', 'report', 'TEXT')
+    env.cmd('FT.ADD', 'incidents', 'doc1', 1.0, 'FIELDS', 'report', 'report content')
+    env.cmd('FT.DICTADD', 'slang', 'timmies', 'toque', 'toonie', 'serviette', 'kerfuffle', 'chesterfield')
+    rv = env.cmd('FT.SPELLCHECK', 'incidents', 'qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq',
+        'TERMS', 'INCLUDE', 'slang', 'TERMS', 'EXCLUDE', 'slang')
+    env.assertEqual([0], rv)
```

Comment:
the return reply is : 
```
test:testIssue501
30: 	❌  (FAIL):	[0] == [['TERM', 'qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq', []]]	test.py:1899
```
This is why the tests failing.

Reply:
fixed. ramp is unable to package

Reply:
I handling the RAMP issue now, you will need to pull from master after I will fix it. 
see PR : https://github.com/RedisLabsModules/RediSearch/pull/503

Reply:
sure thing